Source/DataSources/GeoJsonDataSource.js

```diff
@@ -821,11 +819,7 @@ define([
 
         var promise = data;
         options = defaultValue(options, defaultValue.EMPTY_OBJECT);
-        var sourceUri = options.sourceUri;
         if (typeof data === 'string') {
-            if (!defined(sourceUri)) {
-                sourceUri = data;
```

Comment:
Add this line back in.  Just instead of checking `if (!defined(sourceUri))` it should just always be `sourceUri = data`

Reply:
Hi @hpinkos, if I do this, sourceUri will be an unresolved global variable.  Is this the desired functionality?  I'm not sure how to proceed without getting this warning.

Reply:
Declare `var sourceUri` outside of the if blokc

Reply:
Ok, thank you.  The only issue now is that my editor is giving me a warning about sourceUri being an unused variable.  Should I not worry about this?

Reply:
Did you remember to pass it into the `load` function?